apps/cookbook/src/app/examples/dropdown-example/examples/attention-level.ts

```diff
@@ -2,20 +2,69 @@ import { Component, Input } from '@angular/core';
 
```

Comment:
All the changes in this file are just to have a good showcase for the three different versions of dropdown on each theme. It is not the most beautiful solution, but at least it mirrors 100% how we do it for button.